cmd/commands/root.go

```diff
@@ -23,6 +23,7 @@ type rootFlags struct {
 	namespace   string
 	minSeverity string
 	exitCode    int
+	includegenerated bool
```

Comment:
Can we make this camel case to be consistent?
```suggestion
	includeGenerated bool
```